### PR TITLE
Private surveys

### PR DESCRIPTION
**Pull request by DavidAndreev**

### CODE REVIEW

File: indico/modules/events/surveys/templates/management/email_dialog.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{% from 'forms/_form.html' import simple_form %}
+
+{{ simple_form(form, submit=_('Send'), disabled_until_change=disabled_until_change) }}
```
Comment (ThiefMaster) [inline indico/modules/events/surveys/templates/management/email_dialog.html]: Why not just pass `false` here? You never set it to true in the controller...

File: indico/modules/events/surveys/__init__.py [hunk 2]
```diff
@@ -115,3 +115,12 @@ class SurveysRole(ManagementRole):
     name = 'surveys'
     friendly_name = _('Surveys')
     description = _('Grants management access to surveys.')
+
+
+@signals.get_placeholders.connect_via('survey-link-email')
+def _get_placeholders(sender, event, survey, **kwargs):
+    from indico.modules.events.surveys.placeholders import (EventTitlePlaceholder, SurveyTitlePlaceholder,
```
Comment (ThiefMaster) [inline indico/modules/events/surveys/__init__.py]: I would do this:

```
from indico.modules.events.surveys import placeholders
yield placeholders.EventTitlePlaceholder
yield placeholders.SurveyTitlePlaceholder
yield placeholders.SurveyLinkPlaceholder
```

File: indico/modules/events/surveys/templates/management/survey.html [hunk 3]
```diff
@@ -1,6 +1,31 @@
 {% extends 'events/surveys/management/_survey_base.html' %}
 
 {% block content %}
+    {% if survey.private and survey.state.name in ('active_and_clean', 'active_and_answered')%}
```
Comment (nop33) [inline indico/modules/events/surveys/templates/management/survey.html]: space missing `%}`

File: indico/modules/events/surveys/templates/management/survey.html [hunk 4]
```diff
@@ -1,6 +1,31 @@
...
+        <div class="action-box highlight">
+            <div class="section">
+                <div class="icon icon-lock"></div>
+                <div class="text">
+                    <div class="label">
+                        {% trans %}The survey is private{% endtrans %}
```
Comment (nop33) [inline indico/modules/events/surveys/templates/management/survey.html]: In some places you use `{%-` and in others (like here) just `{%`. Choose your preferred one and use that :)

File: indico/modules/events/surveys/templates/management/survey.html [hunk 5]
```diff
@@ -1,6 +1,31 @@
...
+    {% if survey.private and survey.state.name in ('active_and_clean', 'active_and_answered') %}
+        <div class="action-box highlight">
+            <div class="section">
+                <div class="icon icon-lock"></div>
+                <div class="text">
+                    <div class="label">
+                        {%- trans %}The survey is private{% endtrans -%}
+                    </div>
+                    {% trans -%}
+                        Only people who received the link can answer the survey.
```
Comment (ThiefMaster) [inline indico/modules/events/surveys/templates/management/survey.html]: We should show the link somewhere so you don't *have to* send emails.

**Comment by mvidalgarcia** [comment]
Superseded by #2873

**Pull request closed without merging**